actions.jsx

```diff
@@ -189,7 +191,39 @@ export function setIndicator(indicator) {
 }
 
 
+export function fetchApplicationBootstrap() {
+  return dispatch => {
+    dispatch(requestApplicationBootstrap());
+    const bootstrapEndpoint = $.ajax({
+      type: 'GET',
+      url: 'https://nxt.staging.lizard.net/bootstrap/kpi/',
+      xhrFields: {
+        withCredentials: true,
+      },
+      success: (data) => {
+        return data;
+      },
+    });
+    Promise.all([bootstrapEndpoint]).then((data) => {
+      console.log('data', data);
```

Comment:
Dit is kunst ;)
Maar nee, het is de Redux action die het `lizard/bootstrap` endpoint bevraagt en in de app state opneemt.